visdag/ops/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file should exist and be empty or have contents, but a single blank line is weird.

Reply:
Emptied file.